pallets/polkadex-ido/src/test.rs

```diff
@@ -700,22 +704,28 @@ fn test_show_interest_in_round_randomized_participants() {
             );
         }
 
+        assert_eq!(
+            PolkadexIdo::register_investor(Origin::signed(6_u64)),
+            Ok(())
+        );
 
-
-
+        // Oversubsribed
+        assert_eq!(
+            PolkadexIdo::show_interest_in_round(
+                Origin::signed(6_u64),
+                round_id,
+                100
+            ),
+            Err(Error::<Test>::NotAllowed.into())
+        );
```

Comment:
I would suggest to move this assertion into a separate test with it's own name to not put all assertions into a single test since purpose of this test, regarding of it's name, is different.

Reply:
So this assertion can take place only if the previous assertions are completed, that is we need to oversubscribe the round and see if it allows any new investors. The name of the test is misleading because previously it was to test random eviction, instead I could change the name of the test itself?

Reply:
If we don't need to test random eviction anymore - I guess defining proper name for test is ok.